namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/K8sDtabStoreInitializer.scala

```diff
@@ -26,3 +26,5 @@ class K8sDtabStoreInitializer extends DtabStoreInitializer {
   override def configClass = classOf[K8sConfig]
   override def configId = "io.l5d.k8s"
 }
+
+object K8sDtabStoreInitializer extends K8sDtabStoreInitializer
```

Comment:
What exactly is the purpose of the empty companion object here?

Reply:
so you don't have to type `new` to use it :)

Reply:
rather, so you have a static instance on which you can reference its members without allocating a new initializer instance